### ISSUE TITLE
Mix persisted and transient state within component?

### ISSUE THREAD

**Issue by pke** [tonyhb/redux-ui#20]
I have a component that needs to persist some state and some can be forgotten when its unmounted. Is that currently supported? I don't see any way to specify that.

**Comment by tonyhb** [comment]
Yeah, mixing persistent and transient state isn't currently supported. May I ask why we need to save some but not the other? Interested to hear what this is for. Hopefully we can find a plan that works with the current implementation.

If not we'll think about how to support this.
